extensions/shell-buddy/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "shell-buddy",
+  "title": "Shell Buddy",
+  "description": "Converts natural language prompts to shell commands with AI",
+  "icon": "command-icon.png",
+  "author": "jebraat",
+  "categories": [
+    "Developer Tools",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Convert Prompt",
+      "subtitle": "GPT-3",
```

Comment:
GPT-3?

Reply:
I interpreted the subtitle to name the "service it integrates with". The command conversion uses GPT-3 AI under the hood. I renamed it to "Shell Buddy" - I think that makes more sense anyway :)


<img width="873" alt="Xnapper-2022-10-11-08 57 37" src="https://user-images.githubusercontent.com/36050523/194943278-a86abc33-39b1-46cf-b79c-b49f5cb3e878.png">